catalog/utilities/media_props_gen/docs/media_props.md

```diff
@@ -2,424 +2,376 @@
-
-## Existing Data Inconsistencies
+The URL of the creator's page, usually on the source site.
 
 # title
 
 ## Description
 
-## Object Shape
+The title of the media item.
 
-## Selection Criteria
+## Existing Data Inconsistencies
 
-## Normalization and Validation
+Provider scripts may include html tags in record titles, see
+[issue #1441](https://github.com/WordPress/openverse/issues/1441)
 
-## Existing Data Inconsistencies
+Some Wikimedia titles in the database still include "FILE:" prefix, and
+unnecessary file extension, which is
+[hot-fixed](https://github.com/WordPress/openverse/blob/3329d307f5cdce0cbe1f1c13dbe17106970abdaa/frontend/src/utils/decode-media-data.ts#L50)
+in the frontend. Some titles were incorrectly decoded, for which there is a
+hot-fix in the frontend
 
 # meta_data
 
 ## Description
 
-## Object Shape
+A JSON object containing additional metadata about the media item. This must
+contain the `license_url` (automatically added by the `MediaStore` class from
+the `License` object).
 
 ## Selection Criteria
 
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+Relevant information that is not covered by other fields should be added here.
+This includes such items as the dates of creation, publication, geographical
+data, descriptions, and popularity data.
 
 # tags
 
 ## Description
 
+The list of tags associated with the media item.
+
 ## Object Shape
 
-## Selection Criteria
+jsonb array of dictionaries: `{"name": "tag1", "provider": "wordpress"}`. Some
+tags are provider-generated, and include an `accuracy` field with a float value.
+
+If there are no tags, the field should be set to null, not an empty array or
+empty object.
 
 ## Normalization and Validation
 
-## Existing Data Inconsistencies
+Tags are cleaned in the `MediaStore` class. The tags that contain license slugs
+are removed because they are often misleading, using a different license than
+the media item itself (see
+https://github.com/cc-archive/cccatalog-api/issues/253).
 
-# watermarked
+## Existing Data Inconsistencies
 
-## Description
+The cleanup process in data refresh fixes the following tag inconsistencies:
 
-## Object Shape
+- Empty tags (`{}`) are filtered out, see
+  [cc-archive issue](https://github.com/cc-archive/cccatalog-api/issues/130)
+- Tags with license slugs are removed because they often don't match the media
+  item license. These tags, as well as some other deny-listed tags are filtered
+  out.
+- Some machine-generated tags have accuracy lower than 90% and are unreliable.
+  These tags are filtered out.
 
-## Selection Criteria
+Some inconsistencies are not fixed by the cleanup process:
 
-## Normalization and Validation
+- Incorrectly encoded tags, see
+  [issue #1927](https://github.com/WordPress/openverse/issues/1927). This can
+  result in duplicate tags when the frontend decodes the tags.
+- Tags with leading or trailing spaces, see
+  [issue 4199](https://github.com/WordPress/openverse/issues/4199)
```

Comment:
Nit for consistency:
```suggestion
  [issue #4199](https://github.com/WordPress/openverse/issues/4199)
```

I believe these can also _just_ be referenced as `#4199` - our documentation building plugin should link them appropriately to GitHub!